src/inquire.js

```diff
@@ -34,6 +34,16 @@ const TECH_STACK = {
   ]
 };
 
+const USE_HTTPS = {
+  type: 'list',
+  name: 'useHttps',
+  message: 'Please chose a protocol for connecting to github',
+  choices: [
+    { name: 'https: ( https://github.com/yourOrg/yourRep )', value: true },
+    { name: 'ssh: ( git@github.com:yourOrg/yourRepo )', value: false }
```

Comment:
@JaKXz thoughts on this ? https://github.com/rangle/rangle-starter/issues/5#issuecomment-200419688